siliconcompiler/utils/multiprocessing.py

```
@@ -199,41 +199,52 @@ def stop() -> None:
+                    # Try without the lock
+                    if manager.__board:
+                        manager.__board.stop()
+                        manager.__board = None
 
-        # Remove the log file if the run was successful
-        if not manager.__error:
+            if manager.__manager_server:
+                # Shut down the multiprocessing manager
+                MPManager.__address = None
+                manager.__manager.shutdown()
+        except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)